api/data_explorer/test/test_export_url_controller.py

```diff
@@ -0,0 +1,33 @@
+import json
+
+from data_explorer.test.base_test_case import BaseTestCase
+
+dataset_name = 'Test data'
```

Comment:
All config variables should be set in app.config.update. Delete this line and change below:
```
        app.config.update({
            'DATASET_NAME': 'Test data',
            'TABLE_NAMES': ['project_id.dataset_id.table_name']
        })
```